### ISSUE TITLE
Caffeine Tile generating background data on Android 13?

### ISSUE THREAD

**Issue by xanoni** [otaconix/caffeinetile#7]
Hey there, the project looks dead, so I'm not sure if I'm gonna receive a response, but let's try anyway...

I noticed in the app settings that Caffeine Tile had generated a few kB of background data on one of my vanilla Android 13 phones. See screenshot below (mobile background data switch was deactivated only after I noticed this, but afaik this doesn't solve the problem, as the app can still use WiFi background data).

![image](https://user-images.githubusercontent.com/77220130/216657267-7bdac3cb-2fda-4437-91ff-8b5342db85a3.png)

Version: 1.3 (latest on F-Droid, added on Jan 21st, 2017)

What is this about? I didn't expect it to make any network connections, became paranoid, and restored my phone from a backup ;-) Please tell me I just overreacted.

**Comment by otaconix** [comment]
That is very, very strange! I can assure you it is definitely not supposed to use any data whatsoever. If you can read the source code, I'd encourage you to do that, you should see it does not use any networking. 

That said, I just reinstalled the app myself, and can't seem to reproduce the issue. Maybe you could try disabling the use of background data for the app? In fact you could disable the use of both mobile and wifi.

I'll keep looking for a way to fully disable any kind of networking on my end, but can't make any promises regarding a fix for now. Thanks for reporting this!

**Comment by xanoni** [comment]
Yeah, indeed strange ... this package has been sitting in F-Droid for years, so I'd guess this doesn't happen very often, or one of the other tinfoil heads would have noticed it before me.

I'm not sure how reproducible this issue is. But if someone was very motivated to look into this, I'm sure they could learn something interesting about Android and how it does the traffic volume accounting :-)
